openeogeotrellis/utils.py

```diff
@@ -572,3 +574,85 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 @functools.lru_cache
 def is_package_available(name: str) -> bool:
     return any(m.name == name for m in pkgutil.iter_modules())
+
+
+def reproject_cellsize(
+        spatial_extent: dict,
+        native_resolution: dict,  # cell_width, cell_height, crs
+        to_crs: str,
+) -> Tuple[float, float]:
+    """
+    :param spatial_extent: Cellsize conversion could be different when done at the poles or the equator
```

Comment:
this param description is a bit confusing

Reply:
Is this ok?
 ```
The spatial extent is needed, because conversion is often
     different when done at the poles compared to the equator.
     eg: When converting 1meter to degrees (in LatLon) at the North Pole, it can be 14% degrees more in LatLon
     compared to the same conversion at the equator.
```

Reply:
I think it can even be more than 14%:  the longitude degrees per meter ratio goes to infinity at the poles

Reply:
I made it more general. When I did the calculation, I checked how much the Y size changed. The X-size seems to change 9x when going at the north side. However, the UTM zone I used to check only went till 85deg to the north pole